usermods/rc_switch_v2/rc_switch_v2.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include "wled.h"
+
+#include <RCSwitch.h>
+
+#ifndef USERMOD_RC_SWITCH_PIN
+#ifdef ARDUINO_ARCH_ESP32
+#define USERMOD_RC_SWITCH_PIN 22
```

Comment:
Not sure which pins to use?

Reply:
Check online resources to determine available GPIO.
One would be RandomnerdTutorials.com